code/data_structures/src/stack/infix_to_prefix/infix_to_prefix.cpp

```diff
@@ -27,12 +26,12 @@ int getPriority(char c)
 
 /** Function that converts infix
     expression to prefix expression. **/
-string infixToPrefix(string infix)
+std::string infixToPrefix(std::string infix)
 {
     // stack for operators.
-    stack<char> operators;
+    std::stack<char> operators;
     // stack for operands.
-    stack<string> operands;
+    std::stack<string> operands;
```

Comment:
string is undefined. use std::string. There are many other places where you have done the same mistake. Do correct it.